docs/3.0/develop/variables.mdx

```diff
@@ -68,77 +68,93 @@ In addition to the UI and API, you can reference variables in code and in certai
 
 You can interact with variables through the Python SDK using the `get`, `set`, and `unset` methods. 
 
-Outside of a `flow` or `task`, these methods are always invoked asynchronously.
-
-#### In an asynchronous context
 ```python
-from prefect import flow
 from prefect.variables import Variable
 
-@flow
-async def my_async_flow():
-    # set a variable
-    answer = await Variable.set(name="the_answer", value="42")
-    print(answer) # 42
-    
-    # get a variable
-    answer = await Variable.get("the_answer")
-    print(answer) # 42
-    
-    # overwrite an existing variable
-    answer = await Variable.set(name="the_answer", value="43", overwrite=True)
-    print(answer) # 43
-    
-    # unset a variable
-    await Variable.unset("the_answer")
-    
-    # get a variable that doesn't exist
-    answer = await Variable.get("not_the_answer")
-    print(answer) # None
-    
-    # get a variable that doesn't exist with a default
-    answer = await Variable.get("not_the_answer", default="42")
-    print(answer) # 42
+assert Variable.set("answer", 42) == Variable(name="answer", value=42, tags=[])
+
+assert Variable.get("answer") == 42
+
+assert Variable.set("answer", 9001, overwrite=True) == Variable(name="answer", value=9001, tags=[])
+
+assert Variable.unset("answer") is True
+
+assert Variable.get("answer", "fallback") == "fallback"
 ```
 
-While inside a `flow` or `task`, variables should be invoked synchronously or asynchronously 
-depending on whether the current context is synchronous or asynchronous.
+You can use `overwrite=True` to update the value of an existing variable.
+
+<Warning>
+**Contextual Behavior**
+
+In a sync context (like an `if __name__ == "__main__"` block or simple `def` scope), these methods are used synchronously.
```

Comment:
```suggestion
In a sync context, such as an `if __name__ == "__main__"` block or basic definition scope, these methods are used synchronously.
```

Reply:
updated in [4e4fc1c](https://github.com/PrefectHQ/prefect/pull/15233/commits/4e4fc1cc53077728a2389521aa49dce19e6160a6)